EnglishFilter/sections/antiadblock.txt

```diff
@@ -574,6 +574,10 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80755
+||pagead2.googlesyndication.com/getconfig/sodar$xmlhttprequest,redirect=nooptext,domain=jaanmusic.in
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=jaanmusic.in
```

Comment:
added a bit different, with redirect for adsbygoogle.js Current solution is not working at the moment.